docs/docs-content/vm-management/vm-migration-assistant/migrate-vms-vmo-cluster.md

```diff
@@ -0,0 +1,929 @@
+---
+sidebar_label: "Migrate VMs to a VMO cluster"
+title: "Migrate VMs to a VMO cluster"
+description: "Learn how to migrate VMs to Palette VMO using the VM Migration Assistant"
+icon: " "
+hide_table_of_contents: false
+sidebar_position: 20
+tags: ["vmo", "vm migration assistant"]
+---
+
+Follow this guide to create source providers and migrate your VMs to your VMO cluster.
+
+## Limitations
+
+- You can only migrate VMs hosted in VMware vSphere.
```

Comment:
I'd suggest to have some versions of vSphere that we support. Let me check in the slack channel what we should do for that.